wandb/integration/diffusers/resolvers/sdxl.py

```diff
@@ -0,0 +1,268 @@
+import logging
+from typing import Any, Dict, List, Sequence
+
+import torch
```

Comment:
like the util comment make sure that this requirement is guaranteed to be installed and if not let the user know

Reply:
Keeping numpy and torch imports outside the scope of functions because they're also being used for type annotation in a few cases.

Reply:
if you need for type annotation you can just import it as part of the `TYPE_CHECKING` condition

Reply:
Done